src/content/docs/products/splashkit/SplashKit Tutorials/Onboarding/01-windows-installation.md

```diff
@@ -0,0 +1,194 @@
+---
+title: Windows Installation Guide
+description: Use this template to create a proposal for a new SplashKit tutorial.
+sidebar:
+  order: 1
+  label: 1. Windows Installation
+  attrs:
+    class: windows
+---
+
+This is a guide on how to install Splashkit on Windows, along with Visual Studio Code and the necessary extensions required to run Splashkit. This guide is intended to get you set up and ready to contribute to the SplashKit team if you're a Windows user. This installation guide was done on Windows 10.
+
+## Install msys2
+
+### Download msys2
+
+To start, head to the official [msys2 website](https://www.msys2.org/) and download the installer.
+
+![msys2](/splashkit/windows-installation-fig1.png)
+
+### Install msys2
+
+When the installer is downloaded, run it, you will get a pop up that looks like the following:
+
+![msys2](/splashkit/windows-installation-fig2.png)
+
+You will need to click `more info` then click run. The following shows the screenshots of each step and the options selected:
+
+
+![msys2](/splashkit/windows-installation-fig4.png)
+
+![msys2](/splashkit/windows-installation-fig5.png)
+
+![msys2](/splashkit/windows-installation-fig6.png)
+
+![msys2](/splashkit/windows-installation-fig7.png)
+
+## Install Command Line tools
+
+Copy and paste the following into your new **MINGW64** terminal:
+
+```bash
+pacman -S git mingw-w64-x86_64-clang mingw-w64-x86_64-gcc mingw-w64-x86_64-gdb --noconfirm --disable-download-timeout
+```
+
+You will need to right click in the terminal and select paste as the terminal does not support `ctrl + v`.
+
+![msys2](/splashkit/windows-installation-fig8.png)
+
+![msys2](/splashkit/windows-installation-fig9.png)
+
+## Installing SplashKit
+
+### Collect the Installation scripts for SplashKit
+
+Open your **MSYS2** terminal and run the following command to download the SplashKit installation scripts:
+
+```bash
+bash <(curl -s https://raw.githubusercontent.com/splashkit/skm/master/install-scripts/skm-install.sh)
+```
+
+![msys2](/splashkit/windows-installation-fig10.png)
+
+### Execute skm
+
+**Restart** your terminal, then execute the following:
+
+```bash
+skm
+```
+
+Then the command:
+
+```bash
+skm global install
+```
+
+![msys2](/splashkit/windows-installation-fig11.png)
+
+## Install Visual Studio Code
```

Comment:
Are we limited to only using VSCode or can we use any IDE?

Reply:
Yes, unless someone else wants to add in others. But the default is typically VSCode.

Reply:
I think you're missing the intent of my comment as a rhetorical. 

It may be worth noting in the guide that any IDE can be used, however VSCode is the default/recommended, with the associated guide.